tests/Unit/TestHelpers.hpp

```diff
@@ -114,9 +114,9 @@ template <typename T, Requires<tt::has_equivalence<T>::value> = nullptr,
 void test_move_semantics(T&& a, const T& comparison, Args&&... args) {
   static_assert(std::is_rvalue_reference<decltype(a)>::value,
                 "Must move into test_move_semantics");
-  static_assert(std::is_nothrow_move_assignable<T>::value,
+  static_assert(std::is_move_assignable<T>::value,
```

Comment:
why did you change these?

Reply:
Ah, perhaps this should have been a different commit.
The reason is that some of the types now tested with this are not nothrow assignable and this assert fails. Do you prefer creating another utility to support those types?

Reply:
The types only copy pointers, why should that ever throw?

Reply:
I agree this should work. As far as I can tell, it is entirely related to the storage order change, the `is_nothrow_move_assignable` succeeds when `blaze::defaultStorageOrder` is used, and fails now that I've defaulted it to `columnMajor`.
It also appears that this problem existed previously - if you change the default storage order to `blaze::columnMajor` in develop and try to compile with Blaze 3.2, it fails the same nothrow assert. I haven't been able to track down precisely _why_ it behaves in this way, but I can only imagine that it is a subtle Blaze inheritance defect.
How would you like me to proceed for this?